README.md

```diff
@@ -265,15 +265,15 @@ tokenManager: {
 }
 ```
 
-By default, the `tokenManager` will attempt to renew expired tokens. When an expired token is requested by the `tokenManager.get()` method, a renewal request is executed to update the token. If you wish to manually control token renewal, set `autoRenew` to false to disable this feature. You can listen to  [`expired`](#tokenmanageronevent-callback-context) events to know when the token has expired.
+By default, the `tokenManager` will attempt to renew tokens before they expire. If you wish to manually control token renewal, set `autoRenew` to false to disable this feature. You can listen to [`expired`](#tokenmanageronevent-callback-context) events to know when the token has expired.
```

Comment:
This contradicts the CHANGELOG

Reply:
Can you clarify where the confusion/contradiction is?

Reply:
This says tokenManager will attempt to renew tokens.  The CHANGELOG says that tokenManager.get() no longer does, but doesn't mention that the renewal ability still exists.

Reply:
OK, I will update CHANGELOG to re-terate that renew is being done by tokenManager (outside the get()) method.